transports/file/src/test/resources/file-age-config.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:file="http://www.mulesoft.org/schema/mule/file"
+      xsi:schemaLocation="
+       http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+       http://www.mulesoft.org/schema/mule/file http://www.mulesoft.org/schema/mule/file/current/mule-file.xsd">
+    <file:connector name="fileConnector" autoDelete="true" streaming="true" validateConnections="true" pollingFrequency="60000" fileAge="${fileAgeConnector}"/>
+    <flow name="testageFlow">
+        <file:inbound-endpoint name="fileEndpoint1" path="src/main/resources/Dir1" responseTimeout="10000" connector-ref="fileConnector"/>
+        <file:outbound-endpoint path="src/main/resources/Archive" outputPattern="#[message.inboundProperties.originalFilename]" connector-ref="fileConnector" responseTimeout="10000"/>
```

Comment:
Outbound endpoints are not used in the test. Replace them with an echo-component